packages/teleport/src/player/Xterm/Xterm.tsx

```diff
@@ -28,6 +28,12 @@ export default function Xterm({ tty }: { tty: Tty }) {
     });
 
     term.open();
+    term.extendGetBufferCoords(refContainer);
+
+    term.tty.on('terminal.data', () => {
```

Comment:
There is an enum for 'terminal.data'. Lets use that instead of hardcoded string.

Reply:
revised: https://github.com/gravitational/webapps/pull/99/commits/868b747bdb286ec6d6ef2bf64b529a11fe252d6b